src/pages/video-listing-page/VideoListing.jsx

```diff
@@ -0,0 +1,44 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import Aside from "../../components/aside/Aside";
+import "./VideoListing.css";
+export function VideoListing() {
+  const [videos, setVideos] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      const response = await axios.get("/api/videos");
+      console.log(response.data);
+      if (response.status === 200) {
+        setVideos(response.data.videos);
+      }
+    })();
+  }, []);
+
+  return (
+    <div className="trend-video-grid-layout">
+      <Aside />
+      <main className="trend-main">
+        <h2 className="trend-title">Trending Videos</h2>
+        <div className="video-container">
+          {videos.map((video) => (
+            <div className="card card-vrt" key={video._id}>
```

Comment:
suggestion - you can extract video card as a separate component

Reply:
will separate out soon as files changes